src/components/app/MobileWorkBoard.jsx

```
@@ -10,10 +9,12 @@ import viewPink from '../../assets/icons/common/common_view_pink.svg'
 import bookmarkGray from '../../assets/icons/common/common_bookmark_gray.svg'
 import bookmarkPink from '../../assets/icons/common/common_bookmark_pink.svg'
 import TagIcon from '../web/TagIcon'
+import { useNavigate } from 'react-router-dom'
```

ESLint

[error] 'useNavigate' is defined but never used.

(no-unused-vars)